robolectric/build.gradle

```diff
@@ -45,5 +45,5 @@ dependencies {
     testImplementation "androidx.test:runner:$axtRunnerVersion@aar"
     testImplementation libs.guava
     testCompileOnly AndroidSdk.MAX_SDK.coordinates // compile against latest Android SDK
-    testRuntimeOnly AndroidSdk.MAX_SDK.coordinates // run against whatever this JDK supports
-}
\ No newline at end of file
+    testRuntimeOnly files("${System.getenv("ANDROID_HOME")}/platforms/android-34/android.jar")
```

Comment:
Maybe we should define a variable in Gradle with the compile SDK version, so we can use it everywhere, and in particular her.
I have a feeling it will be rather easy to forget to update this value when compile SDK is changed (I suppose that's why 34 is used?).

Reply:
Yeah definitely a good idea, this PR is still a WIP, just trying to see if this will cause any issues.